cmake/modules/3.20/FindPostgreSQL.cmake

```diff
@@ -177,6 +177,7 @@ endif()
 function(__postgresql_find_library _name)
   find_library(${_name}
    NAMES ${ARGN}
+   NAMES_PER_DIR
```

Comment:
would you mind contributing that upstream to https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/FindPostgreSQL.cmake?ref_type=heads#L183 ?

Reply:
On a second look, this spot is not relevant (but doesn't do harm) because it is not multi-name. Upstream anyways, or revert?

Reply:
better not change if not relevant